game_eggs/steamcmd_servers/dont_starve/egg-don-t-starve-together.json

```diff
@@ -15,9 +15,9 @@
         "quay.io\/parkervcp\/pterodactyl-images:debian_source"
```

Comment:
this should be `ghcr.io/parkervcp/steamcmd:debian` but that image is not yet public